jupyterhub/handlers/base.py

```diff
@@ -431,9 +431,15 @@ def get_next_url(self, user=None):
             # default URL after login
             # if self.redirect_to_server, default login URL initiates spawn
             if user and self.redirect_to_server:
-                next_url = user.url
+                if self.config.JupyterHub.get('default_user_page'):
+                    next_url = self.config.JupyterHub['default_user_page']
+                else:
+                    next_url = user.url
             else:
-                next_url = url_path_join(self.hub.base_url, 'home')
+                if self.config.JupyterHub.get('default_page'):
+                    next_url = self.config.JupyterHub['default_page']
```

Comment:
Certainly can add it. I need to get my cluster running on the latest hub before I can validate any of this. But you are suggesting ``next_url = url_path_join(self.hub.base_url, self,config.JupyterHub['default_page'].

Also, I *think* in the current implementation unless default_page is like /user/<xxz>. The proxy will redirect to base_url anyway, but I'm all for leaving out obscure subtlety if it makes things clearer.

However, I was going to adopt @minrk's nomenclature of default_url rather than default_page. Is it just me or would 'url' connote a full path rather than path minus the base_url. Maybe it's just me.